.github/workflows/advanced.yml

```diff
@@ -307,6 +307,7 @@ jobs:
       - name: Patch toolchain stage2
         if: ${{ steps.cache-toolchain.outputs.cache-hit != 'true' && matrix.platform == 'pc-cygwin' }}
         run: |
+          RESET_SOURCES=${{ matrix.platform == 'pc-cygwin' && '1' || '0' }}
```

Comment:
What about moving it in the config?

Reply:
Yes, that would be better.

Reply:
I am not sure why it cannot be configured in the config.sh based on the target.

Reply:
If it would be set in `config.sh`, it would be force both for stage 1 and stage 2 patching. It needs to be forced only for stage 2.

Reply:
Anyway it is preferable to have it in the script to have similar experience with local env.